devicon.json

```diff
@@ -2039,27 +2039,11 @@
                 "plain"
```

Comment:
In the line 2035, do you think you could add some tags?

Reply:
Well, it's a company that does lots of IDEs, so I added `ide`, couldn't think of anything else.